schema/deploy/tables/review_comment_001.sql

```diff
@@ -0,0 +1,35 @@
+-- Deploy ggircs-portal:tables/review_comment_001 to pg
+-- requires: tables/review_comment
+
+begin;
+
+alter table ggircs_portal.review_comment drop column form_id;
+alter table ggircs_portal.review_comment add column application_review_step_id int references ggircs_portal.application_review_step(id);
+create index ggircs_portal_review_comment_app_review_step_foreign_key on ggircs_portal.review_comment(application_review_step_id);
+comment on column ggircs_portal.review_comment.application_review_step_id is 'Foreign key to application_review_step. Defines the review step that this comment belongs to.';
+
+create or replace function ggircs_portal_private.create_review_steps()
+  returns void as $$
+declare
+  temp_row record;
+  temp_id int;
+begin
+    for temp_row in
+      select id from ggircs_portal.application
+    loop
+      insert into ggircs_portal.application_review_step(application_id, review_step_id)
+      values (temp_row.id, 1)
```

Comment:
Where is this review step created? I was expecting to see it in `tables/application_review_step` but I don't

Reply:
This function does two things: creates legacy application_review_steps for all existing applications and attaches all comments for the application to that legacy step. I suppose it could be two functions, one in the `application_review_step` migration to create the legacy steps and another here in this migration to connect all comments to the legacy `application_review_step`. I can split this function up to make sure the data is migrated more clearly.

All new application_review_steps are created via a trigger on application submit.

Reply:
I was confused where the `1` was coming from, but that's from the `review_step` table, not `application_review_step`.
Maybe replace the `1` with `(select id from ggircs_portal.review_step where step_name = 'legacy')` to make it explicit?

Reply:
In view of @kriscooke 's comment in https://github.com/bcgov/cas-ciip-portal/pull/1574#issuecomment-816927193, maybe the `step_name` should be `other` instead of `legacy`

Reply:
:+1:  I can make both those changes